### PR TITLE
pythonPackages.aioinflux: init at 0.9.0

### PR DESCRIPTION
**Pull request by lopsided98**
###### Motivation for this change

Add the aioinflux Python package, an asyncio client for InfluxDB. This is an updated version of https://github.com/NixOS/nixpkgs/pull/70840. @liamdiprose I have included you as a maintainer, as in your original PR. Let me know if you would rather not be a maintainer for this package.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 111849` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python37Packages.aioinflux</li>
    <li>python38Packages.aioinflux</li>
  </ul>
</details>

File: pkgs/development/python-modules/aioinflux/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
```
Comment (fabaff) [inline pkgs/development/python-modules/aioinflux/default.nix:3]: Not used.

File: pkgs/development/python-modules/aioinflux/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+buildPythonPackage rec {
+  pname = "aioinflux";
+  version = "0.9.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1jy5mcg9wdz546s9wdwsgkxhm2ac4dmphd9vz243db39j1m0a3bj";
+  };
+
+  propagatedBuildInputs = [ aiohttp ciso8601 pandas ];
```
Comment (fabaff) [inline pkgs/development/python-modules/aioinflux/default.nix:19]: `pandas` is one of the [`extras_require`](https://github.com/gusutabopb/aioinflux/blob/master/setup.py#L43-L53). 

One opinion is to not add `extras_require` (#62833).

**Review by fabaff — COMMENTED** [review]
- [x] package path fits guidelines
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.platforms` is set
- [x] `meta.maintainers` is set
- [x] package build (see https://github.com/NixOS/nixpkgs/pull/111875#issuecomment-773149664)
- [ ] all depending packages build
- [x] library tested on x86_64

```bash
...ew/pr-111849/results/python38Packages.aioinflux/lib/python3.8/site-packages]$ python
Python 3.8.7 (default, Dec 21 2020, 17:18:55) 
[GCC 10.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from aioinflux import InfluxDBClient
>>> client = InfluxDBClient(host='192.168.1.123', db='nixos', mode='blocking')
>>> client.ping()
{'Content-Type': 'application/json', 'Request-Id': '8df43c61-66d2-11eb-8001-2aee9ed82338', 'X-Influxdb-Build': 'OSS', 'X-Influxdb-Version': '1.8.4', 'X-Request-Id': '8df43c61-66d2-11eb-8001-2aee9ed82338', 'Date': 'Thu, 04 Feb 2021 10:20:07 GMT'}
[...]
```

Looks good to me.

**Comment by fabaff** [comment]
> 2 packages built:

Hmm, the package for Python 3.9 is missing.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 111849` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python37Packages.aioinflux</li>
<li>python38Packages.aioinflux</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
> > 2 packages built:
> 
> Hmm, the package for Python 3.9 is missing.

```
error: boto-2.49.0 not supported for interpreter python3.9
```

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 111849` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python37Packages.aioinflux</li>
<li>python38Packages.aioinflux</li>
</ul>
</details>

File: pkgs/development/python-modules/aioinflux/default.nix (hunk 2 above)
Comment (lopsided98) [reply pkgs/development/python-modules/aioinflux/default.nix:18]: I kept that based on the discussion here: https://github.com/NixOS/nixpkgs/pull/70840#discussion_r344992370, but I don't really care one way or the other.

Comment (SuperSandro2000) [reply pkgs/development/python-modules/aioinflux/default.nix:18]: If it is a package you normally want to use just add it.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 111849` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python37Packages.aioinflux</li>
<li>python38Packages.aioinflux</li>
</ul>
</details>

**Pull request merged**